tests/integration/test_join.py

```diff
@@ -110,6 +110,24 @@ def test_join_right(c):
     assert_eq(return_df, expected_df, check_index=False)
 
 
+def test_join_cross(c, request):
```

Comment:
Any reason why we're getting the tables here using `request` instead of directly specifying them as fixtures to use? Typically we've only used `request` when we want to parametrize by whether or not we're using GPU, in which case the signature would look something like:

```python
@pytest.mark.parametrize(
    "user_table,department_table",
    [
        ("user_table_1", "department_table"),
        pytest.param("gpu_user_table_1", "gpu_department_table", marks=pytest.mark.gpu),
    ],
)
def test_join_cross(c, user_table, department_table, request):
```